hs_tracking/views.py

```diff
@@ -4,9 +4,39 @@
 from django.views.generic import TemplateView
 from django.utils.decorators import method_decorator
 from django.contrib.auth.decorators import user_passes_test
-from django.http import HttpResponse
+from django.http import HttpResponse, HttpResponseRedirect
 
 from . import models as hs_tracking
+from .models import Session, Variable
+from .utils import get_std_log_fields
+
+
+class AppLaunch(TemplateView):
```

Comment:
Just to be clear, are you suggesting that I invoke a signal that handles the logging logic from the a redirectview class?  

I'm not familiar with the RedirectView class (https://docs.djangoproject.com/en/1.11/ref/class-based-views/base/) but I'll give it a shot.